hardhat/tasks/func-mintBridgedToken.js

```diff
@@ -0,0 +1,52 @@
+const { readLastDeployedAddress } = require("../helpers/fileHelpers");
+const { display } = require("../helpers/loggingHelper");
+
+/**
+ * @dev caller MUST BE the admin / deployer
+ */
+task("func-mintBridgedToken", "mint amount of bridged token to user")
+  .addParam("to", "The address of the user")
```

Comment:
```suggestion
  .addParam("to", "The user's address")
```

Reply:
I avoid the lonely quotes because it breaks the formatting.

We will check